localization/react-intl/src/app/components/media/MediaActionsMenuButton.json

```diff
@@ -1,30 +1,32 @@
 [
   {
     "id": "mediaActions.assignOrUnassign",
+    "description": "Menu item to select a team to assign item",
     "defaultMessage": "Assign to…"
   },
   {
     "id": "mediaActions.unlockStatus",
+    "description": "Menu item to unlock an item status so it can be changed",
     "defaultMessage": "Unlock status"
   },
   {
     "id": "mediaActions.lockStatus",
+    "description": "Menu item to lock an item status so it cannot be changed",
     "defaultMessage": "Lock status"
   },
   {
     "id": "mediaActions.sendToTrash",
+    "description": "Menu item to move the current item to the trash list",
     "defaultMessage": "Send to Trash"
   },
   {
     "id": "mediaActions.sendToSpam",
+    "description": "Menu item to move the current item to the spam list",
     "defaultMessage": "Mark as Spam"
   },
   {
     "id": "mediaActions.history",
+    "description": "Menu item to view the history of changes to the item",
     "defaultMessage": "Item history"
-  },
-  {
-    "id": "mediaActions.tooltip",
```

Comment:
@amoedoamorim here is one that was removed as noted in my other comment

Reply:
yeah, that's probably happening because of the inline passing of the message descriptor object. Please use the `defineMessages` import from `react-intl` to generate the strings correctly for translation!

Reply:
got it! thanks @amoedoamorim , I updated the strings